app/backend/approaches/chatapproach.py

```diff
@@ -114,6 +124,7 @@ async def run_with_streaming(
         followup_questions_started = False
         followup_content = ""
         async for event_chunk in await chat_coroutine:
+            print(event_chunk, flush=True)
```

Comment:
This was probably used for debugging purposes. If it makes sense printing something we should at least be using the configured logger.